styles/globals.css

```diff
@@ -1,3 +1,10 @@
+:root{
```

Comment:
[qq] Not sure what is best practice for CSS.
It seems unmanageable to have one MASSIVE global.css file. 

I imagine it'd be better to split it up so one file describes the css of one component

Thoughts?

Reply:
Idk lol